chapters/fr/chapter1/1.mdx

```diff
@@ -0,0 +1,51 @@
+
+**Matthew Carrigan** est un ingénieur en Machine Learning chez Hugging Face. Il vit à Dublin, en Irlande et à travaillé auparavant comme ingénieur en ML chez Parse.ly et avant cela comme chercheur postdoctoral à Trinity College Dublin. Il ne croit pas que nous arrivions à AGI en mettant à l'échelle les architectures existantes, mais il a tout de même beaucoup d'espoir dans l'immortalité des robots.
+
+**Lysandre Debut** est un ingénieur en Machine Learning chez Hugging Face et a travaillé sur la librairie 🤗 Transformers depuis les premières phases de développement. Son but est de rendre NLP accessible pour tout le monde en développant des outils disposant d'une API très simple.
+
+**Sylvain Gugger** est un ingénieur en recherche chez Hugging Face et un des principaux responsable de la librairie 🤗 Transformers. Précédemment, Il a été chercheur en ML chez fast.ai et a écrit _[Deep Learning for Coders with fastai and PyTorch](https://learning.oreilly.com/library/view/deep-learning-for/9781492045519/)_ avec Jeremy Howard. Son but ultime est de rendre le deep learning plus accessible, en développant et en améliorant des techniques permettant aux modèles d'apprendre rapidement sur des ressources limitées.
+
+**Merve Noyan** est développeuse advocate chez Hugging Face et travaille à la création d'outils et de contenu visant à démocratiser le machine learning pour tous.
+
+**Lucile Saulnier** est une ingénieure en Machine Learning chez Hugging Face qui travaille au développement et à l'implémentation de nombreux outils open source. Elle est également activement impliquée dans de nombreux projets de recherche dans le domaine de NLP comme l'entraînement collaboratif de modèles et le projet BigScience.
+
+**Lewis Tunstall** est un ingénieur en Machine Learning chez Hugging Face dévoué au développement d'outils open source avec la volonté de les rendre accessibles à une communauté plus large. Il est également co-auteur d'un livre qui va bientôt paraître, [Transformers for Natural Language Processing](https://www.oreilly.com/library/view/natural-language-processing/9781098103231/).
+
+**Leandro von Werra** est un ingénieur en Machine Learning dans l'équipe open-source chez Hugging Face et également co-auteur du livre qui va bientôt paraître, [Transformers for Natural Language Processing](https://www.oreilly.com/library/view/natural-language-processing/9781098103231/). Il a plusieurs années d'expérience dans l'industrie du machine learning, où il a pu déployer des projets de NLP en production en travaillant sur toutes les étapes clefs du déploiement.
+
+Êtes-vous prêt à commencer ? Dans ce chapitre, vous apprendrez:
+* Comment utiliser la fonction `pipeline()` pour résoudre des problèmes de NLP comme la génération de texte et la classification
+* Quelle est l'architecture d'un modèle Transformer
+* Comment faire la distinction entre les différentes architectures d'encodage, de décodage et d'encodage-décodage et leurs condition d'utilisation
```

Comment:
`encoder`, `decoder` and `encoder-decoder` could stay in english version instead of translating them like that. I don't know what is the best translation to give to that kind of architectures.

Reply:
I chatted with the French speakers at Hugging Face and they suggested using `encodeur`, `decodeur` and `encodeur-decodeur`. What do you think?

Reply:
It seems better 100%